cliquet/__init__.py

```diff
@@ -13,7 +13,8 @@
 from cliquet import utils
 from cliquet.initialization import (  # NOQA
     initialize, initialize_cliquet, install_middlewares)
-
+from cliquet import resource
```

Comment:
Here it is not what we want.

We should probably define __all__ to define the exaustive list of what we want to expose in cliquet itself.

See https://github.com/django/django/blob/master/django/core/files/__init__.py#L3